### DISCUSSION TITLE
Options list not populating

### DISCUSSION THREAD

**Discussion by snvgglebear** [Budibase/budibase#4756] [Help]
I followed this guide to create dynamic filtering of a table, and the options list will not populate with the data from the chosen column. 
I am using postgresql on the backend, and I set the type to options in the data view of Budibase.
https://docs.budibase.com/guides/add-a-dropdown-filter-to-a-table

**Comment by snvgglebear** [comment]
Update: I figured out the problem. When changing the field type to options, the options list does not auto populate. Is there a way to use data from a table column as an option list?

**Comment by gerhardmpl** [comment]
Have a look at #4418 where a bbdb table is used for the options values. The concept can be adapted for option picker that must be the same throughout the app without the need to change column field options. You could also see my post and example app in #4721.

**Reply by snvgglebear** [reply]
That helps, but I am still stuck on how to use the choice to filter a data provider based on the selected dropdown value.

**Comment by gerhardmpl** [comment]
A word of warning: I am new to `Budibase`, so my approach might not be the best one and I am not 100% sure that I understand your problem. Happy to have a Contributor or Maintainer comment because I think that the guide does not work as described - but I could be completely wrong. Just testing my understanding of `Budibase` ;-)

Attached please find an example I put together with _countries_ and _cities_ in those _countries_. Navigate to the `/filter/` screen and you can choose which countries cities you want to see in the table. Pay attention to the `Country Option Picker` _Field_ and _Options source_ as well as to the _Filter_ of the `Data Provider`.

[filter-export-1646247186979.txt](https://github.com/Budibase/budibase/files/8172190/filter-export-1646247186979.txt)
